tests/ext/drv/test_ext_drv.c

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/*++
+
+Abstract:
+WDF based driver that does the following:
+
+Registers as an eBPF program information provider and hook provider.
+
+Environment:
+
+    Kernel mode
+
+--*/
+
+// ntddk.h needs to be included first due to inter header dependencies on Windows.
+#include <ntddk.h>
+
+#pragma warning(push)
+#include <wdf.h>
+
+#include "ebpf_platform.h"
+
+#include "test_ext.h"
+#include "test_ext_helpers.h"
+#include "test_ext_ioctls.h"
+
+#define TEST_EBPF_EXT_DEVICE_NAME L"\\Device\\TestEbpfExt"
+#define TEST_EBPF_EXT_SYMBOLIC_DEVICE_NAME L"\\GLOBAL??\\TestEbpfExtIoDevice"
+
+// Driver global variables
+static DEVICE_OBJECT* _test_ebpf_ext_driver_device_object;
+static BOOLEAN _test_ebpf_ext_driver_unloading_flag = FALSE;
+
+//
+// Pre-Declarations
+//
+static EVT_WDF_IO_QUEUE_IO_DEVICE_CONTROL _test_ebpf_ext_driver_io_device_control;
+DRIVER_INITIALIZE DriverEntry;
+
+static VOID
```

Comment:
```suggestion
static void
```
use standard types, not upper case aliases to them